pkg/controller/directimagemigration/imagestream.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2020 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package directimagemigration
+
+import (
+	"context"
+	//  liberr "github.com/konveyor/controller/pkg/error"
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+	imagev1 "github.com/openshift/api/image/v1"
+	kapi "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func (t *Task) getDirectImageStreamMigrations() ([]migapi.DirectImageStreamMigration, error) {
+	ismList := migapi.DirectImageStreamMigrationList{}
+	err := t.Client.List(
+		context.TODO(),
+		k8sclient.MatchingLabels(t.Owner.GetCorrelationLabels()),
+		&ismList)
+	if err != nil {
+		return nil, err
```

Comment:
I wonder whats the value in sending a nil return value, worst case if a developer does not guard against that, the code will panic. I would think `return ismList, err` is a better way, the user of this function will simple check for error and iterate on the list. 

Ohh that also reminds me, we need to replace all `return err` with `return liberr.Wrap(err)`?

Reply:
From what I remember, this seems to be what most of the kubernetes code I've interacted with does. Maybe there are other patterns as well. I know that, in general, if something returns an item and an error, if the error isn't nil, my code calling it can't rely on the item existing, and it will usually be ignored anyway. I guess there's no harm in just returning it, but the inconsistency might cause confusion. As for `liberr.Wrap(err)`, the function that this is returning to (in task.go) does that wrapping, so I didn't think it was needed here.